school-library/students/student.rb

```diff
@@ -0,0 +1,12 @@
+require '../person/persons'
+
+class Student < Animal
+  def initialize(classroom)
```

Comment:
- [ ] The constructor parameters of the `Student` class should also include all parameters previously in the `Person` class. .e.g. `def initialize(age, classroom, name = "Unknown", parent_permission = true)`. Kindly fix this